packages/core/useScroll/index.ts

```diff
@@ -0,0 +1,121 @@
+import { ref, reactive } from 'vue-demi'
+import { useThrottleFn, useDebounceFn, noop } from '@vueuse/shared'
+import { MaybeElementRef } from '../unrefElement'
+import { useEventListener } from '../useEventListener'
+
+export interface UseScrollOptions {
+  /**
+   * Throttle time for scroll event,it’s disabled by default.
+   *
+   * @default 0
+   */
+  throttle?: number
+
+  /**
+   * The check time when scrolling ends.
+   * This configuration will be setting to (throttle + checkStopTime) when the `throttle` is configured.
+   *
+   * @default 200
+   */
+  checkStopTime?: number
+
+  /**
+   * Trigger it when scrolling ends.
+   *
+   */
+  onStop?: () => void
+
+  /**
+   * Enabled direction for scroll.
+   *
+   * @default ['x','y']
+   */
+  enabledDirection?: Array<'x'|'y'>
```

Comment:
When will this be used? Or I think we can just do it for both without the config, there should be much perf difference

Reply:
By default, we do both. i think we should expose this config and let ppl setting it  to save perf overhead

Reply:
I don't see there is any overhead on that. It's just variable assignment, and we don't have extra event listener.

Reply:
Okay,Do you suggest that we remove this config and enable both by default ?

Reply:
removed~